lms/static/js/ccx/view/schedule_right_container.js

```diff
@@ -0,0 +1,221 @@
+var define = window.define || RequireJS.define;  // jshint ignore:line
+
+define(
+  [
+    'backbone',
+    'jquery',
+    'underscore',
+    'gettext',
+    'text!templates/ccx/underscore/right-container.underscore',
+    'text!templates/ccx/underscore/form.underscore',
+    'js/vendor/timepicker/jquery.timepicker'
+  ],
+  function (
+    Backbone,
+    $,
+    _,
+    gettext,
+    scheduleRightContainerTemplate,
+    scheduleFormTemplate
+  ) {
+    'use strict';
+    return Backbone.View.extend({
+
+      events: {
+        'change select#ccx_chapter': 'onChapterSelect',
```

Comment:
Nit: we generally avoid using IDs and rely on classes instead. IDs can easily be problematic because they must be unique within the page.